easybuild/easyconfigs/x/XALT/XALT-2.10.15.eb

```diff
@@ -0,0 +1,60 @@
+name = 'XALT'
+version = '2.10.15'
+
+homepage = 'https://github.com/xalt/xalt'
+description = """XALT 2 is a tool to allow a site to track user executables
+  and library usage on a cluster. When installed it can tell a site what are
+  the top executables by Node-Hours or by the number of users or the number
+  of times it is run. XALT 2 also tracks library usage as well. XALT 2 can
+  also track package use by R, MATLAB or Python. It tracks both MPI and
+  non-MPI programs."""
+
+toolchain = SYSTEM
+
+# The location of XALT configuration file must be specified via
+# 'config_py' easyconfig parameter.  You can edit the easyconfig file,
+# or use 'eb --try-amend=config_py=<path>'. " See
+# https://xalt.readthedocs.io/en/latest/030_site_filtering.html for
+# more information.
+config_py = None
```

Comment:
we don't need to specify a config file ?

Reply:
I tried to keep it as similar as possible to the [XALT-2.8.4.eb recipe](https://github.com/ComputeCanada/easybuild-easyconfigs/blob/computecanada-main/easybuild/easyconfigs/x/XALT/XALT-2.8.4.eb). The options would be specified with the `--try-amend` option when building.

Reply:
For our purposes, we prefer either be explicit in the EasyConfig recipe, or use hooks to change the values
https://github.com/ComputeCanada/easybuild-computecanada-config/blob/main/cc_hooks_gentoo.py

In the hooks, you would need to add the specific configuration parameters in the list here: 
https://github.com/ComputeCanada/easybuild-computecanada-config/blob/main/cc_hooks_gentoo.py#L717
and then add a directory entry for XALT here: 
https://github.com/ComputeCanada/easybuild-computecanada-config/blob/main/cc_hooks_gentoo.py#L613

Reply:
Specifying the options explicitly would be easier to implement and understand in my opinion. Where would be the best location for the config file?

Reply:
Yes, that is correct

Reply:
then add it in the `sources` list, put it in the git repository alongside the EasyConfig, and use `%(builddir)s/config.py` as the location